pkg/server/registry/apigroups/acorn/volumes/translator.go

```diff
@@ -22,6 +22,33 @@ type Translator struct {
 }
 
 func (t *Translator) FromPublicName(ctx context.Context, namespace, name string) (string, string, error) {
+
+	i := strings.LastIndex(name, ".")
+	// If there is not a period, parse it not as an alias
+	if i == -1 || i+1 > len(name) {
```

Comment:
I don't understand the point of the `i+1 > len(name)` part of this. If the `.` is the last character in the string, then `i+1 == len(name)`. I don't think it can possibly be greater than the length of the name, unless I'm mistaken about something.

Reply:
Talked with Darren about it. Supposed to check the last character. Bug in indexing. Fixing this